src/serviceWorker.js

```diff
@@ -11,17 +11,17 @@
 // opt-in, read https://bit.ly/CRA-PWA
```

Comment:
The changes in this files have been made by eslint so it just include cosmetics.